__tests__/pages/_app.test.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import {
+  render,
+  waitFor,
+  waitForElementToBeRemoved
+} from '@testing-library/react'
+import { MockedProvider } from '@apollo/client/testing'
+import MyApp from '../../pages/_app'
+import Login from '../../pages/login'
+import * as Sentry from '@sentry/browser'
+import GET_APP from '../../graphql/queries/getApp'
+import dummyLessonData from '../../__dummy__/lessonData'
+import dummySessionData from '../../__dummy__/sessionData'
+import dummyAlertData from '../../__dummy__/alertData'
+import posthog from 'posthog-js'
+jest.spyOn(Sentry, 'captureException')
```

Comment:
It's better to mock the whole posthog module here instead of that single function.

```js
jest.mock('posthog-js')
import posthog from 'posthog-js'
```

Then you can check with the real function name.

```js
expect(posthog.init).not.toHaveBeenCalled()
```